detox/package.json

```diff
@@ -52,7 +52,7 @@
     "e2e:android-test-release": "detox test -c android.emu.release --record-logs failing --take-screenshots failing",
     "e2e:ios-test": "IOS=true detox test -c ios.sim.debug",
     "e2e:ios-build-release": "detox build -c ios.sim.release",
-    "e2e:ios-test-release": "IOS=true detox test -c ios.sim.release --headless --record-logs failing --take-screenshots failing",
+    "e2e:ios-test-release": "IOS=true detox test -c ios.sim.release --record-logs failing --take-screenshots failing",
```

Comment:
Reverted the change here since it will be useful in local dev to see action in headed mode. Local or CI may run `npm run e2e:ios-test-release -- --headless` if need to run in headless. cc @metanerd for pipeline update.